### PR TITLE
BaSP-TG-66: Get and delete employee

### PR DESCRIPTION
**Pull request by camilavaleriarojas**
Logic to get and delete the data from the back-end using redux was implemented, and IsLoading spinner shared component was applied.

### CODE REVIEW

File: src/Components/Employees/index.js [hunk 1]
```diff
@@ -44,8 +34,9 @@ function Employees() {
     <>
       <div className={styles.container}>
         <h1>Employees</h1>
+        <Spinner isLoading={isLoading} />
```
Comment (lmalvasia) [inline src/Components/Employees/index.js:37]: When the isLoading is true, we should render only the spinner. In this case we are rendering all the components and I think that this is not the best UI experience 😂

Comment (camilavaleriarojas) [reply src/Components/Employees/index.js:37]: You're right, my bad! thanks

File: src/redux/employees/actions.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+export const getEmployeesError = (error) => {
+  return {
+    type: GET_EMPLOYEES_ERROR,
+    payload: error
+  };
+};
+
+export const deleteEmployeesPending = () => {
```
Comment (baudraccoverona) [inline src/redux/employees/actions.js:30]:
```suggestion
export const deleteEmployeePending = () => {
```

File: src/redux/employees/actions.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  return {
+    type: DELETE_EMPLOYEES_PENDING
+  };
+};
+
+export const deleteEmployeesSuccess = (payload) => {
```
Comment (baudraccoverona) [inline src/redux/employees/actions.js:36]:
```suggestion
export const deleteEmployeeSuccess = (payload) => {
```

File: src/redux/employees/constants.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+export const GET_EMPLOYEES_PENDING = 'GET_EMPLOYEES_PENDING';
+export const GET_EMPLOYEES_SUCCESS = 'GET_EMPLOYEES_SUCCESS';
+export const GET_EMPLOYEES_ERROR = 'GET_EMPLOYEES_ERROR';
+export const DELETE_EMPLOYEES_PENDING = 'DELETE_EMPLOYEES_PENDING';
```
Comment (baudraccoverona) [inline src/redux/employees/constants.js:4]:
```suggestion
export const DELETE_EMPLOYEE_PENDING = 'DELETE_EMPLOYEES_PENDING';
```

Comment (baudraccoverona) [inline src/redux/employees/constants.js:4]: Same as mentioned above for all the delete constants

**Review by lmalvasia — APPROVED** [review]

**Review by lmalvasia — APPROVED** [review]

**Pull request merged**